packages/manager/apps/pci-public-ip/src/api/hooks/useFailoverIP.ts

```diff
@@ -0,0 +1,73 @@
+import { PaginationState } from '@ovhcloud/manager-components';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { FailoverIP, Instance } from '@/interface';
+import { getAllFailoverIP } from '../data/failover-ip';
+import { useAllInstance } from './useInstance';
+import { paginateResults } from '../utils/pagination';
+
+export type FailoverOptions = {
+  pagination: PaginationState;
+};
+
+const aggregateFailoverIPs = (
+  failoverIPData: FailoverIP[],
+  instanceData: Instance[],
+) => {
+  const aggregatedData = failoverIPData.map((failover) => {
+    const failoverEdited = { ...failover };
+
+    let instance = {} as Instance;
+
+    if (failover.routedTo) {
+      instance = instanceData.find(({ id }) => id === failover.routedTo);
+    }
+
+    return {
+      ...failoverEdited,
```

Comment:
failoverEdited can be replaced with failover since there are no edits made

Reply:
well spotted! fix done.